package.json

```diff
@@ -79,9 +78,6 @@
   },
   "devDependencies": {
     "@semantic-release-plus/docker": "^3.1.0",
```

Comment:
you can remove this too, and refresh package-lock

Reply:
@derberg Done 👍 . Sorry for the delay